src/pages/BuildingDetailPage/BuildingDetailPage.jsx

```
@@ -12,6 +12,7 @@ import {
     FloorList,
     HeaderArea,
     MainViewer,
+    PanelHint,
```

ESLint

[error] 'PanelHint' is defined but never used.

(no-unused-vars)